snippet-template.md

```diff
@@ -23,6 +23,12 @@ class ComponentName extends React.Component {
 ReactDOM.render(<ComponentName />, mountNode);
```

Comment:
Accidentally commited this on this PR's branch, instead of master, but it's not a problem I hope (as the need for a notes section appeared in this PR).